workshop/posting-error-example.md

```diff
@@ -0,0 +1,201 @@
+
+*For our example:*  
+<img src = "screenshots/what-is-the-code.png" width = "600">
+
+### What is the error or problematic outcome?  
+
+- Include any error messages verbatim. Also preferably not a screenshot so others can copy-paste for Google searching.
+- If you don't have an error message per se, but do have something else that is not working as expected, describe what isn't working and how you found it.
+     - *Example*: "This data.frame I'm showing in my code, called `cool_df` doesn't have row names when I look at it in my environment panel".
+- Context around when the error appears versus when it does not can be helpful clues to your reader.
+
+*For our example:*  
+<img src = "screenshots/what-is-the-error.png" width = "600">
+
+### (Optional) What have you tried thus far?  
+
+- This is considered an optional point because we understand you also may not know where to begin (and that is definitely okay - we are here to help!)
+
+- If you have tried some things to fix the problem; it can be helpful for others to know what you've tried and narrow down on what the problem might be.
+
+- If you have tried solutions you have found posted online, including the links to those solutions is helpful.
+
+*For our example:*  
+<img src = "screenshots/what-have-you-tried.png" width = "600">
+
+## Step 3) Post to Slack!
+
+Navigate to your particular workshop's training channel.
+
+If you wrote out our example question post in a text editor, it may have looked like this; where backticks will become code chunks after we copy-paste it in Slack:
+
+<img src = "screenshots/text-editor.png" width = "500">
+
+When you are typing out your question you may find it helpful to have our "four points" as your headers but that is up to you.
+We've also included [a checklist you could use for creating your post](#posting-a-code-question-checklist).
+
+When we copy and paste our text in Slack, it may ask  you if you'd like to `Apply formatting`. Click `Apply`:
+
+<img src = "screenshots/slack-apply-formatting.png" width = "500">
+
+This will change your post to be formatted:
+
+<img src = "screenshots/slack-formatting-applied.png" width = "500">
+
+If all looks set, click the green arrow to post it!
+
+**Slack Tip:** if you see a mistake in your post you missed, you can click on the three dots in the corner of your Slack post and choose `Edit message` and fix the mistake:
+
+<img src = "screenshots/slack-edit-message.png" width = "500">
+
+## Step 4) Look for responses (in the same thread)!
+
+We don't want anyone's questions or responses to get lost, especially after you've followed these careful steps to craft it!
+
+So to keep responses from getting lost, we strongly encourage you to make your main question one post and keep all further messages and correspondence about that question in the same Slack thread.
```

Comment:
I think you should hit this point harder. From #11, emphasis added here:

> Threading - both when first asking the question and **when returning to the same notebook a few steps later with a new question** (helps with context if a different instructor answers the question and in a way that requires less explanation on the participant's part!)

The idea behind that write up is that they don't have to type out what notebook they're using again because it will be earlier in the thread. Also, I think sometimes earlier errors can be informative for instructors when debugging (see point about order mattering).